ProjectTemplates/dotnet-new-efprovider-synergy/SynergyEfProvider/SynergyEfProvider.synproj

```diff
@@ -20,13 +20,13 @@
       <Version>6.0.153</Version>
```

Comment:
We also need to update HarmonyCore related references to version 8 when it's released.